docs/modules/servers/pages/distributed/self-hosting.notes

```diff
@@ -0,0 +1,96 @@
+The name of the project is "Jame Self Hosting Sandbox", and I see the term "self hosting" in a few places... So it sounds like it is important, but what is that intended to mean?
```

Comment:
Distributed server is not intended for self-hosting.

Hence I would discuss this elsewhere. (mailing list?)

Reply:
I can't remember why I copied these notes here, but in any case you bring up a good point. It is important we mention for whom this server is intended, and why they may want to use it.